index.html

```diff
@@ -15,5 +15,107 @@
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
 
+
+<header>
+    <div id="header">
+        <div class="logo">
+            <img src="favicon.ico" alt="logo Karma"/>
+        </div>
+
+        <nav>
+            <div id="nav-bar">
```

Comment:
You may use an `ul` element here instead of a `div` and have the child `button` elements as `li` elements. 
This is a common structure used for a list of links.